Components/TeamList.js

```diff
@@ -0,0 +1,78 @@
+import useSWR from "swr";
+import styled from "styled-components";
+
+const Header = styled.h1`
+  font-size: 5rem;
+  font-weight: bold;
+  margin-top: 1rem;
+  color: #000000; // #0d48a0 (Tory Blue / h2)
```

Comment:
I think black is the default color anyway so you do not have to assign it here? But in any case you could delete the comment about h2

Reply:
I was playing around with this headline - now removed.